rpc/bidderapi/v1/bidderapi.proto

```diff
@@ -126,19 +140,25 @@ message Commitment {
   int64 block_number = 3 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
     description: "Max block number that the bidder wants to include the transaction in."
   }];
-  string received_bid_digest = 4 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+  uint64 decay_start_timestamp = 4 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+    description: "Timestamp at which the bid starts decaying."
+  }];
+  uint64 decay_end_timestamp = 5 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+    description: "Timestamp at which the bid ends decaying."
+  }];
+  string received_bid_digest = 6 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
     description: "Hex string encoding of digest of the bid message signed by the bidder."
   }];
-  string received_bid_signature = 5 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+  string received_bid_signature = 7 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
```

Comment:
It's typically safer to append new pb fields to the end of a `message` s.t if a node expects the old byte schema, the existing fields will be deserialized correctly, and the new fields will be blank (if empty/null is even allowed)

Reply:
Updated